packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-in/nodejs/integrationsteps.md

```diff
@@ -1,2 +1,111 @@
+
+```JavaScript
+router.post('/login', async (req, res, next) => {
+  const { username, password } = req.body;
+  const authClient = getAuthClient(req);
+  const transaction = await authClient.idx.authenticate({
+    username,
+    password,
+  });
+  handleTransaction({ req, res, next, authClient, transaction });
+});
+```
+
+### Step 3: Handle the response from the sign in
+
+The application handles the response from the authentication call using the `handleTransaction` function as shown in the SDK sample application `handleTransaction.js` file. The `transaction` parameter is the `IdxStatus` passed in through the response from Okta.
+
+```JavaScript
+module.exports = function handleTransaction({
+  req,
+  res,
+  next,
+  authClient,
+  transaction,
+}) {
+  const {
+    nextStep,
+    tokens,
+    status,
+    error,
+  } = transaction;
+
+...
+```
+
+#### Success status
+
+For a successful sign-in response, the `IdxStatus` field indicates a success `IdxStatus.SUCCESS` and retrieves the token and processes the authenticated user in the app. The SDK sample application
+saves the tokens to storage in the `handleTransaction.js` file and redirects back to the home page.
+
+```JavaScript
+case IdxStatus.SUCCESS:
+      // Save tokens to storage (req.session)
+      authClient.tokenManager.setTokens(tokens);
+      // Redirect back to home page
+      res.redirect('/');
+      return;
+
+...
+
+```
+
+#### Other status
+
+You need to handle other returned `IdxStatus` cases if the user didn't sign in successfully. For example, in the SDK application's `handleTransactions.js` file:
+
+```JavaScript
+ switch (status) {
+    case IdxStatus.PENDING:
+      // Proceed to next step
+      try {
+        if (!proceed({ req, res, nextStep })) {
+          next(new Error(`
+            The current flow cannot support the policy configuration in your org.
+          `));
+        }
+      } catch (err) {
+        next(err);
+      }
+      return;
+    case IdxStatus.SUCCESS:
+      // Save tokens to storage (req.session)
+      authClient.tokenManager.setTokens(tokens);
+      // Redirect back to home page
+      res.redirect('/');
+      return;
+    case IdxStatus.FAILURE:
+      authClient.transactionManager.clear();
+      next(error);
+      return;
+    case IdxStatus.TERMINAL:
+      redirect({ req, res, path: '/terminal' });
+      return;
+    case IdxStatus.CANCELED:
+      res.redirect('/');
+      return;
+  }
+
+```
+
+### Step 4: Get user profile information-optional
+
+Optionally, you can obtain basic user information after the user is authenticated by making a request to Okta's Open ID Connect authorization server. See [Get user profile information after sign in](/docs/guides/oie-embedded-sdk-alternate-flows/java/main/#getuserprofileinfo).
```

Comment:
```suggestion
Optionally, you can obtain basic user information after the user is authenticated by making a request to Okta's Open ID Connect authorization server. See [Get user profile information after sign-in](/docs/guides/oie-embedded-sdk-alternate-flows/java/main/#getuserprofileinfo).
```

Reply:
Again, will leave this for now as is to match other languages.